tox.ini

```diff
@@ -49,6 +54,36 @@ basepython =
     pypy: pypy
     flake8,flakeplus,apicheck,linkcheck,configcheck,pydocstyle: python2.7
 
+[testenv:2.7-integration-sqs]
```

Comment:
If anyone knows a way to rewrite these `tox` sections in a less repetitive way, please let me know.